api/api_test.go

```diff
@@ -252,6 +256,7 @@ func (s *apiSuite) TestNotFoundError(c *gc.C) {
 func (s *apiSuite) TestSignedAgreementsEnvTermsURL(c *gc.C) {
 	cleanup := testing.PatchEnvironment("JUJU_TERMS", "http://example.com")
```

Comment:
Consider moving these *EnvTermsURL tests to a separate test suite? They share common setup.